metricflow/cli/dbt_connectors/adapter_backed_client.py

```diff
@@ -198,9 +204,11 @@ def dry_run(
             f"\n\n{indent_log_line(stmt)}\n"
             + (f"\nwith parameters: {dict(sql_bind_parameters.param_dict)}" if sql_bind_parameters.param_dict else "")
         )
-        # TODO - rely on self._adapter.dry_run() when it is available so this will work for BigQuery and Databricks
-        # without any gymnastics.
-        if self.sql_engine_type is SqlEngine.DATABRICKS:
+        # TODO - consolidate to self._adapter.validate_sql() when all implementations will work from within MetricFlow
+        if self.sql_engine_type is SqlEngine.BIGQUERY:
+            with self._adapter.connection_named(f"Metricflow_BigQuery_dry_run_attempt_{start}"):
```

Comment:
If the name will be visible from the logs, making `start` human-readable would be helpful.com

Reply:
Helpful.com? LOL.

Agreed, there's a separate PR out to clean that up.